Tests/Form/DataTransformer/ModelToIdPropertyTransformerTest.php

```diff
@@ -195,7 +195,7 @@ public function testTransformMultiple()
         $collection[] = $entity2;
         $collection[] = $entity3;
 
-        $this->modelManager->expects($this->exactly(3))
+        $this->modelManager->expects($this->exactly(6))
```

Comment:
The `expects` call issues an assert expecting the precise number of calls to occur which means the test would fail without the changed value.